client/my-sites/plugins/plugins-browser-list/style.scss

```diff
@@ -31,8 +31,8 @@
 	justify-content: space-between;
 	flex-wrap: wrap;
 
-	padding-top: 48px;
 	.plugins-browser-list__titles {
+		padding-top: 48px;
```

Comment:
Might have to do this another way. I'm assuming this is the cause behind this alignment regression:

![Screenshot(2)](https://user-images.githubusercontent.com/811776/185540815-436310b1-e5e1-4c93-abcb-de8ae9d3698f.png)